fluent-dom/src/localization.js

```diff
@@ -268,32 +244,29 @@ function messageFromContext(ctx, errors, id, args) {
  * @param {Array<string>}  keys
  * @param {{Array<{value: string, attrs: Object}>}} translations
  *
- * @returns {Boolean}
+ * @returns {Set<string>}
  * @private
  */
 function keysFromContext(method, ctx, keys, translations) {
   const messageErrors = [];
-  let hasErrors = false;
+  let missingIds = null;
```

Comment:
The docstring says `keysFromContext` returns a Set but this is set to `null` here.